boa/src/lib.rs

```diff
@@ -89,7 +89,8 @@ pub fn forward_val(engine: &mut Interpreter, src: &str) -> Result<Value> {
         Ok(expr) => expr.run(engine),
         Err(e) => {
             eprintln!("{}", e);
-            std::process::exit(1);
+            panic!(); // Panic instead of exit means that tests will continue.
+                      // std::process::exit(1);
```

Comment:
I don't think we need to explain this.
```suggestion
            panic!();
```

Reply:
This is left over from when I did this before we added it to master hence the extra comment